client/gutenberg/extensions/presets/jetpack/editor-shared/render-svg.jsx

```diff
@@ -0,0 +1,10 @@
+/** @format */
+
+const renderSVG = svg => (
+	<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+		<path fill="none" d="M0 0h24v24H0V0z" />
+		{ svg }
+	</svg>
+);
```

Comment:
Not a blocker, but i think instead of putting this in the jetpack preset, we could put it `gutenberg/extensions/shared` i think it is a helpful util for any block in any preset!

Reply:
`gutenberg/extensions/shared` will also be included in view files which is not what we want. 
I will move it to `gutenberg/extensions/utils` since that seems to be a better place.